Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyExport.Codeunit.al

```diff
@@ -199,4 +208,16 @@ codeunit 30284 "Shpfy Company Export"
     begin
         CreateCustomers := NewCustomers;
     end;
+
+    local procedure GetShpfyPaymentTermsIdFromCustomer(Customer: Record Customer; var ShpfyPaymentTermsId: BigInteger): Boolean
```

Comment:
nit: why doesn't this procedure just return the Id?

Reply:
FillInShopifyCompany() procedure receives CompanyLocation that already has "Shpfy Payment Terms Id" value.

New value for CompanyLocation."Shpfy Payment Terms Id" has to be assigned only if Customer Payment Terms Code has value in Shpfy Payment Terms mapping.

If we gonna implement GetShpfyPaymentTermsIdFromCustomer procedure in the way to return Id, it gonna overwrite "Shpfy Payment Terms Id" value to 0 (if mapping is not detected).

I think the task that we have to trigger updates only if mapping exist.